### DISCUSSION TITLE
práctica Pastelería (¿Cómo ven mi práctica?)

Use @media, flex-box y css-grid junto con sass (Cuando Sass entra a tu vida no lo dejas de usar 🥺)

### DISCUSSION THREAD

**Discussion by Isma47** [Launch-X-Latam/MisionFrontEnd#268] [Show and tell]
Página web (cliente)
https://preeminent-daffodil-9baa3f.netlify.app/index.html

Pagina web (Admin)
https://delightful-rolypoly-1adff0.netlify.app/

**Comment by doguedogue** [comment]
Genial 🚀

**Comment by AlvaroFierro** [comment]
Hola muy buena pagina, te esta quedando muy bien
Solamente checa tu footer y el header en mobile, se desborda un poco

**Reply by Amhedriel** [reply]
Si, queda algo desbordado en ambos, checalo por si acaso Isma
